x/subscription/README.md

```diff
@@ -0,0 +1,202 @@
+# `x/subscription`
+
+## Abstract
+
+This document specifies the subscription module of Lava Protocol.
+
+The subscription module is responsible for managing Lava's consumer subscription.  
+To use Lava, consumers must purchase a subscription.  
+The subscription operates on a monthly basis, starting from the next block after purchase and ending one month later.  
+If a user purchases a subscription for more than one month, it will reset at the end of each month until its expiration.  
```

Comment:
> why write reset here and explain it a sentence later?

What do you mean by that?

> I think you should note that the subscription has different limitations enforced by the effective policy. One of the limitations is the allocated monthly CU. When a month passes, the monthly CU counter is reset (or something like that)

How about this:
```
Subscriptions are subject to limitations set by the current policy, including a cap on monthly Compute Units (CUs).  
These CUs reset with each new subscription month, ensuring users get a fresh allocation based on their plan.
```

Reply:
the change you suggested look good

I meant that you write `If a user purchases a subscription for more than one month, it will reset at the end of each month until its expiration.` and then in another sentence: `When a subscription is reset, the monthly left CUs is set to the plan's monthly CU.`

So i didnt understand why didn't you just write something like: "in the end of the month, the monthly left CU is reset" which is simpler

Reply:
Got it, how's this:
>If a user purchases a subscription for more than one month, it will reset at the end of each month until its expiration, with the monthly remaining Compute Units (CUs) being reset to the plan's monthly CU allocation.

Reply:
yeah, that's good

Reply:
Awesome, fixed in 151e51f0462996dd83f355d61c69c3b84947dab5

Reply:
Awesome, fixed in 151e51f0462996dd83f355d61c69c3b84947dab5